integration-test/shadow/shadow_system_test.c

```diff
@@ -740,105 +887,25 @@ void tearDown( void )
 /* ========================== Test Cases ============================ */
 
 /**
- * @brief Subscribes the shadow topics: /update/delta, /update/documents,
+ * @brief Subscribes the classic shadow topics: /update/delta, /update/documents,
  * /update/accepted, /delete/accepted, /get/accepted, then publish the
  * regarding payloads to verify if receiving the notification from the
  * subscribed topics.
```

Comment:
Please also mention about the unsubscribe from delta /update/delta topic.

Reply:
Done.